dwave/system/temperatures.py

```diff
@@ -48,13 +48,13 @@
 def effective_field(bqm,
                     samples=None,
                     current_state_energy=False) -> (np.ndarray,list):
-    '''Returns the effective field for all variables and all samples.
-    
+    r'''Returns the effective field for all variables and all samples.
```

Comment:
Not too fond of the single-quote docstrings, especially mixed with double-quote ones in the same file.

Reply:
I agree, but I wasn't the author of this module, so just did the minimal changes.